File: computer/logger.py
class Logger:
    def error(self, msg: str) -> None:
        print(f"[Error] {msg}")

    def info(self, msg: str) -> None:
        print(f"[Info] {msg}")

    def success(self, msg: str) -> None:
        print(f"[Success] {msg}")

File: computer/server/logger.js
class Logger {
    info = (msg) => {
        console.log(`[Info] ${msg}`);
    };
    success = (msg) => {
        console.log(`[Success] ${msg}`);
    };
    error = (msg) => {
        console.log(`[Error] ${msg}`);
    };
}

module.exports = Logger;
